apps/election-manager/src/utils/semsTallies.ts

```diff
@@ -248,7 +248,9 @@ function sanitizeItem(item: string): string {
 function parseFileContentRows(fileContent: string): SEMSFileRow[] {
   const parsedRows: SEMSFileRow[] = []
   fileContent.split('\n').forEach((row) => {
-    const entries = row.split(',').map((e) => sanitizeItem(e))
+    const entries = row
+      .split(/,(?=(?:(?:[^"]*"){2})*[^"]*$)/)
+      .map((e) => sanitizeItem(e))
```

Comment:
can you expand a bit on what's going on here?

Reply:
oh sorry this is actually unrelated to this PR, I'll at least seperate it to its own clearer commit and see if I can add a unit test. I found this while importing the test sample files in module-converter-sems to generate the new test tallies files there that if there is a comma inside of something like the contest title or candidate name or something that it will not import to election manager properly as before I was just naively splitting on any comma. So:
"contestID","contest name","candidate name","0","12"
worked fine but:
"contestID","contest,name","candidate name","0","12"
would confuse everything. 

This updates the split to only split on commas that are outside of quotation marks so that latter case is handled properly.